core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -308,6 +308,7 @@ private static String toNormalizedIconUrl(String url) {
         icons.addIcon(new Icon("icon-grey icon-sm", "16x16/grey.png", Icon.ICON_SMALL_STYLE));
         icons.addIcon(new Icon("icon-text-error icon-sm", "16x16/text-error.png", Icon.ICON_SMALL_STYLE));
         icons.addIcon(new Icon("icon-text icon-sm", "16x16/text.png", Icon.ICON_SMALL_STYLE));
+        icons.addIcon(new Icon("icon-computer-user-offline icon-sm", "16x16/icon-computer-user-offline.png", Icon.ICON_SMALL_STYLE));
```

Comment:
seems you've just added png everywhere? changelog talks about svg

Reply:
Yeah, the icons weren't added to the icon set registry and therefore weren't overridden by the appropriate SVGs.

Reply:
if they are supposed to be overridden then just remove the .pngs.

Reply:
That'll work too. I wasn't too sure about removing them in favor of compat with older versions.